01 Non blocking/src/js/apiIceAndFire.js

```diff
@@ -19,7 +19,7 @@ const getRequest = (config) => {
 };
 
 const sendGetRequest = (req, url) => {
-    req.open('get', url, true);
+    req.open('get', url, false); // setting async to false, send() does not return until the response is received, then is forced to work synchronously
     req.send();
```

Comment:
No, it's what I am asking for. It's to ensure house before character, but with 'non blocking' style.